mungegithub/mungers/submit-queue.go

```diff
@@ -1636,7 +1653,7 @@ func (sq *SubmitQueue) serveMergeInfo(res http.ResponseWriter, req *http.Request
 	out.WriteString("PRs must meet the following set of conditions to be considered for automatic merging by the submit queue.")
 	out.WriteString("<ol>")
 	if gateCLA {
-		out.WriteString(fmt.Sprintf("<li>The PR must have the label %q or %q </li>", cncfClaYesLabel, claHumanLabel))
+		out.WriteString(fmt.Sprintf("<li>The PR must have the label %q </li>", claYesLabels))
```

Comment:
s/must have the label %q/must have one of the following labels: %q/

Reply:
Done